.changeset/rude-pumpkins-hang.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/core': patch
```

Comment:
This is currently a breaking change right? so `minor`. Perhaps it could be worth making it backwards compatible though, as in `slackChannel?: string | typeof defaultSlackChannel`.